requirements.txt

```diff
@@ -1,2 +1,2 @@
-marshmallow>=2.21,<3.0
+marshmallow>=3.0
```

Comment:
Should likely preserve the upper bound to <4.0 to prevent future breaking changes.